tekton/ci/plumbing-hub-build.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: plumbing-hub-build
+  namespace: tektonci
+spec:
+  params:
+    - name: passorfail
+      description: Should the CI Job 'pass' or 'fail'
+    - name: message
+      description: Message to be logged in the CI Job
+    - name: gitCloneDepth
+      description: Number of commits in the change + 1
+    - name: fileFilterRegex
+      description: Names regex to be matched in the list of modified files
+    - name: checkName
+      description: The name of the GitHub check that this pipeline is used for
+    - name: gitHubCommand
+      description: The command that was used to trigger testing
+  resources:
+    - name: source
+      type: git
+    - name: starttrigger
+      type: cloudEvent
+    - name: endtrigger
+      type: cloudEvent
+  tasks:
+  - name: check-reset
+    taskRef:
+      name: tekton-ci-reset-check-status
+    params:
+    - name: checkName
+      value: $(params.checkName)
+    resources:
+      outputs:
+      - name: trigger
+        resource: starttrigger
+    conditions:
+    - conditionRef: "check-git-files-changed"
+      params:
+      - name: gitCloneDepth
+        value: $(params.gitCloneDepth)
+      - name: regex
+        value: $(params.fileFilterRegex)
+      resources:
+      - name: source
+        resource: source
+    - conditionRef: "check-name-matches"
+      params:
+      - name: gitHubCommand
+        value: $(params.gitHubCommand)
+      - name: checkName
+        value: $(params.checkName)
+  - name: build-hub
+    runAfter: [check-reset]
+    conditions:
+    - conditionRef: "check-git-files-changed"
+      params:
+      - name: gitCloneDepth
+        value: $(params.gitCloneDepth)
+      - name: regex
+        value: "tekton/images/hub/**"
```

Comment:
We should make this an input param of the pipeline, so we could have just one pipeline.

Reply:
Ahhh one pipeline and multiple pipelineruns, I like it, I'll update :wink: